pages/api/self-check/patch.ts

```
@@ -0,0 +1,43 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { SelfCheckGroup } from "../models/selfCheckGroup";
+import { SelfCheckQuestion } from "../models/selfCheckQuestion";
+
+const patchSelfCheckByID = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    // Find self check by id
+    const { id } = req.query;
+    let selfCheck;
+    try {
+        selfCheck = await SelfCheckGroup.findById(id);
+    } catch (err) {
+        console.log("error is ", err);
```

lets return an error here